### ISSUE TITLE
Issue with Stackie when mining stone.

### ISSUE THREAD

**Issue by AndaleTheGreat** [gigaherz/Survivalist#23]
When mining stone quickly there is a crash if Survivlaist and Stackie are installed. 
Thorough testing has shown that it does not occur if either of these is disabled.
Does not seem to work when just throwing the items on the ground. Only when mining.

Crash Report - http://pastebin.com/iQtBupY4
Curse Profile Export - https://drive.google.com/file/d/0B-zCPBgxfnQUNlM1R3Vwd1R5RXc/view?usp=drivesdk
Video of example - https://drive.google.com/file/d/0B-zCPBgxfnQUN2toRWNsdkhSOXM/view?usp=drivesdk

- Forge 10.19.0.2180

- Crafting Tweaks (by BlayTheNinth)
- Ender Compass (by MrIbby)
- Survivalist (by gigaherz)
- Just Enough Items (JEI) (by mezz)
- Sparks Hammers (by Bright_Spark)
- Chameleon (by jaquadro)
- Storage Drawers (by jaquadro)
- Iron Chests (by progwml6)
- ElecCore | Rendering Library (by Elec332)
- CraftingTable IV (by Elec332)
- FastLeafDecay (by Olafskii)
- JourneyMap (by techbrew)
- Simple Grinder (by Rumaruka)
- CXLibrary (by CubeX2)
- More Furnaces (by CubeX2)
- SimpleHarvest (by TehNut)
- LunatriusCore (by Lunatrius)
- Stackie (by Lunatrius)
- Food Expansion (by Lellson)
- Convenient Additions (by NecroticDEV)
- Tree Chopper (by DuchLord)
- OpenBlocks Elevator (by VsnGamer)
- TheDragonLib (by Moritz30)
- ArmorPlus (by Moritz30)
- Reap Mod (by EuhDawson)
- Flour Power (by Anteater46)
- Craftable Saddles (by 456Xander)
- Armor Recalc (by TheOldOne22)
- Level Up! (by BeetoGuy)

Crossposted to Stackie bugs: https://github.com/Lunatrius/Stackie/issues/9

**Comment by gigaherz** [comment]
The crash happens in

```
java.lang.IllegalArgumentException: Stack can not be invalid!
    at net.minecraftforge.oredict.OreDictionary.getOreIDs(OreDictionary.java:437)
    at convenientadditions.item.adventurersPickaxe.OreMagnet.isOreItem(OreMagnet.java:22)
```

So I wonder what's Survivalist's involvement in the issue... :/

**Comment by AndaleTheGreat** [comment]
It was the only two mods I could make do this. I should have pointed out it happens as well with *only* these two mods enabled.
Wasn't sure which caused it but I suspected it was more likely to be a Stackie issue. That is why I posted to both. Lunatrius seems to think it could be fixed at either end (that's how I read it at least) and that he/she will fix it in the next release.
Anyway, thought you'd be interested. Gonna close since it appears there is a solution happening at the other end.
